xml/troubleshooting.xml

```diff
@@ -1079,64 +1079,67 @@
-    no data is lost.
+     
+     If you are experiencing issues with the &gnome; desktop (for
+     example, incorrect screen geometry, blank desktop, etc.), there
+     are several ways to troubleshoot the misbehaving graphical
+     desktop environment. The recommended procedure described below
+     offers the safest option to fix broken &gnome;.
    </para>
-   <procedure xml:id="proc-trouble-login-dk">
-    <step>
-     <para>
-      Switch to a text console by pressing <keycombo>
-      <keycap function="control"/> <keycap function="alt"/> <keycap>F1</keycap>
-      </keycombo>.
-     </para>
-    </step>
-    <step>
-     <para>
-      Log in with your user name.
-     </para>
-    </step>
-    <step>
-     <para>
-      Move the user's &gnome; configuration directories to a temporary
-      location:
-     </para>
-<screen>&prompt.user;mv .gconf  .gconf-ORIG-RECOVER
-&prompt.user;mv .gnome2 .gnome2-ORIG-RECOVER</screen>
-    </step>
-    <step>
-     <para>
-      Log out.
-     </para>
-    </step>
-    <step>
-     <para>
-      Log in again, but do not run any applications.
-     </para>
-    </step>
-    <step>
-     <para>
-      Recover your individual application configuration data (including the
-      Evolution e-mail client data) by copying the
-      <filename>~/.gconf-ORIG-RECOVER/apps/</filename> directory back into the
-      new <filename>~/.gconf </filename> directory as follows:
-     </para>
-<screen>&prompt.user;cp -a .gconf-ORIG-RECOVER/apps .gconf/</screen>
-     <para>
-      If this causes the login problems, attempt to recover only the critical
-      application data and reconfigure the remainder of the applications.
-     </para>
-    </step>
+   <procedure>
+     <title>Troubleshooting &gnome;</title>
+     <step>
+       <para>
+	 Launch &yast; and switch to <guimenu>Security and Users</guimenu>.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Open the <guimenu>User and Group Management</guimenu> dialog and press <guimenu>Add</guimenu>.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Fill out the required fields and press <guimenu>OK</guimenu> to create a new user.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Log out and log in as the new user. This gives with a fresh
+	 installation of &gnome;.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Copy individual &gnome; &gnome;-specific configuration files from the
+	 <filename>.local/.config/</filename> directory of the old
+	 user account to the new one. Log out and log in again as the
+	 new user after every copy operation to check whether &gnome;
+	 still works correctly.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Repeat the previous step until you find the configuration file that
+	 breaks &gnome;.
+       </para>
+     </step>
+     <step>
+       <para>
+	 Log in as the old user, and delete the offending configuration file. Log
```

Comment:
"delete the offending configuration file" sounds dangerous :-) Can we have something like moving the configuration file/directory somewhere else for further analysis (eg. there might be a bug the user wants to report)?

Reply:
Good idea! I'll update the text.